R/utils.R

```diff
@@ -200,3 +200,47 @@ ensure_data.table <- function(data) {
   }
   return(data)
 }
+
+#' @title Print information about a `forecast` object.
+#' @description This function prints information about a forecast object,
+#' including "Forecast type", "Score columns",
+#' "Forecast unit".
+#' @export
+#' @keywords print-forecast
+#' @examples
+#' print(example_quantile %>% as_forecast)
+print.forecast_binary <- function(dat, ...) {
```

Comment:
Methods must have the same arguments as the generic (otherwise R CMD check produces a warning, see https://adv-r.hadley.nz/s3.html#s3-arguments). I therefore suggest changing the function arguments to 

```suggestion
print.forecast_binary <- function(x, ...) {
```

even though x is not really the clearest argument 🤷. But it's enforced by R CMD check.

Reply:
Also I think `...` needs documentation. Suggest something like
```
#' param ... additional arguments for [print()]
```

Reply:
I added several arguments in addition to `...` following `print.scoringutils_check` and done in [d3a75cd](https://github.com/epiforecasts/scoringutils/pull/592/commits/d3a75cda48646d206673faba8ddc90e8510cf536).